plugins/modules/postgresql_info.py

```diff
@@ -1012,6 +1019,13 @@ def get_db_info(self):
 
         for datname in db_dict:
             self.cursor = self.db_obj.reconnect(datname)
+            if self.cursor is None:
+                # that means we don't have permission to access these database
+                db_dict[datname]['namespaces'] = {}
+                db_dict[datname]['extensions'] = {}
+                db_dict[datname]['languages'] = {}
+                db_dict[datname]['error'] = "could not connect to the database (probably permission issue)"
```

Comment:
```suggestion
                db_dict[datname]['error'] = "Could not connect to the database."
```
I suggest avoiding assumptions. Later we could think how to return actual errors.